terraform/environments/development-eu-west-2/devops1/vars

```diff
@@ -0,0 +1,20 @@
+aws_bucket = "development-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-development-terraform-state-london"
+aws_region = "eu-west-2"
+environment = "devops1"
+deploy_to = "development"
+state_prefix = "env:/development"
+workspace_key_prefix = "test-data-generator-stack"
+state_file_name = "test-data-generator.tfstate"
+
+aws_profile = "development-eu-west-2"
+ec2_instance_type = "t3.medium"
```

Comment:
Again I'd specify via a default. We can then override if needed

Reply:
Done. Removed ec2_instance_type from vars.

Reply:
Done. Moved 'ec2_instance_type' and a few other common variables to defaults.